vault/testing.go

```diff
@@ -130,6 +130,7 @@ func TestCoreWithSeal(t testing.T, testSeal Seal, enableRaw bool) *Core {
 
 func TestCoreWithCustomResponseHeaderAndUI(t testing.T, CustomResponseHeaders map[string]map[string]string, enableUI bool) (*Core, [][]byte, string) {
 	confRaw := &server.Config{
+		LogRequestsLevel: "basic",
```

Comment:
This isn't a valid log level, right?

Reply:
But we probably shouldn't do that kind of thing outside of a test specific to the feature.  Unless it's somehow relevant to custom response header behaviour?

Reply:
hmm, make sense.